sonar-db/src/main/java/org/sonar/db/metric/MetricDao.java

```diff
@@ -48,12 +48,7 @@ public MetricDto selectByKey(DbSession session, String key) {
   }
 
   public List<MetricDto> selectByKeys(final DbSession session, List<String> keys) {
-    return DatabaseUtils.executeLargeInputs(keys, new Function<List<String>, List<MetricDto>>() {
-      @Override
-      public List<MetricDto> apply(@Nonnull List<String> input) {
-        return mapper(session).selectByKeys(input);
-      }
-    });
+    return executeLargeInputs(keys, input -> mapper(session).selectByKeys(input));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1612)